chopper_built_value/CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Changelog
 
+## 1.1.0
+
+- Chopper upgraded
```

Comment:
Does this update change the behavior or the API of the library? If not, then I think it's just a `1.0.1`